### ISSUE TITLE
Cannot associate platform to pipeline context

### ISSUE THREAD

**Issue by FabienM** [ovh/cds#2783]
### Through workflow as code / `workflow push`:

```yml
  deploy-playground-eu:
    depends_on:
    - build-image-playground-eu
    when:
    - success
    pipeline: deploy-platform
    application: Lhasa
    platform: containers-dev
```

```
$ cdsctl workflow push URBA .cds/*.yml
$ git diff
```
```diff
diff --git a/.cds/LhasaWorkflow.yml b/.cds/LhasaWorkflow.yml
index fb7fd2b..08cde2f 100644
--- a/.cds/LhasaWorkflow.yml
+++ b/.cds/LhasaWorkflow.yml
@@ -50,7 +50,6 @@ workflow:
     - success
     pipeline: deploy-platform
     application: Lhasa
-    platform: containers-dev
```

### Through UI

Select Workflow / Pipeline / Pipeline context, select `containers-dev` platform and save. CDS replies back with a `400 Bad Request`:

```json
{"message":"Workflow invalide (caused by: Unknown pipeline 6693)"}
```

**Comment by sguiheux** [comment]
Seems to be a problem with pipeline permissions

**Comment by FabienM** [comment]
Partially resolved, replaced by #2825
